### DISCUSSION TITLE
Can we move AA consent later down the funnel?

### DISCUSSION THREAD

**Discussion by ankitagarwaladobe** [ONDC-Official/ONDC-FIS-Specifications#9] [General]
Generally it is seen that having AA consent from the borrower early in the funnel increases the drop in the credit requests.

Can we move it further down the funnel within the select call?

For example:
The user provides all the information (including income details) they have except the AA in the "select call", then the lender responds with the the soft approval and range of offers(amount, tenure, interest rate). 

Now if the user is interested then they can proceed aaccordingly and provide the consent?

Or can it be moved to be part of the search request?

Another optimisation is to take the credit rating from the user itself as an input. This doesn't have to be exact number but a range (example 600-650). The lender can return the response based on that and the can verify later in the next step.

**Comment by Antriksh-Parmar** [comment]
The idea is to provide borrowers with WYSIWYG offers all at once, eliminating the need for multiple back-and-forth interactions to reach the final offer. Lenders will generate the best pre-approved offers possible based on available data.

On the other hand, we would be regressive if we display a static offer to lure borrowers into the funnel. In this scenario, the borrower would need to review each offer and lender individually to find the final offer.

I would appreciate it if you could share the drop-off rate at [antriksh.parmar@ondc.org](mailto:antriksh.parmar@ondc.org). This will help us determine how to further optimize the process.

Regarding your second point, could you please help me understand how this would be an enhancement? Lenders can always perform a soft pull to obtain the exact credit score and generate offers. Introducing another step where the borrower provides a score seems unnecessary, as the lender will eventually need to pull the credit score regardless.

**Reply by nitins11** [reply]
Success rates with AA currently is under 40% as per our discussions with lenders, would like to understand what others are experiencing. Customer journeys today bridge this by having alternate mechanism such as bank statement pdf upload, having WhatsApp chat bots or a dedicated customer service team who would get the application completed. I think just pushing this down the funnel may not help achieve the overall goal. We should consider adding alternative such as pdf upload if AA fails.

**Comment by ankitagarwaladobe** [comment]
**_could you please help me understand how this would be an enhancement?_** 
 
My suggestion was in context to my point 1 where the user can also provide their credit rating range as input with other information like income, kyc etc.

The idea was to provide better search results (static offers) to the users in the first go. The verification of details can then be verified in the next stage where the user provides the consent for AA.

Sure, I will share the drop-off rate over the email.

**Reply by ankitagarwaladobe** [reply]
+ @dilipgoogle
